### DISCUSSION TITLE
Copilot suddenly stopped working

### DISCUSSION THREAD

**Discussion by jekabso21** [community/community#10746] [Copilot Conversations]
When I try to write code the Copilot isn't helping and when I try to press ctrl+Enter it shows that Synthesizing 0/10 solutions. Is it just for me or for other people too?

**Comment by katorly** [comment]
Would it appear suggestions after synthesizing solutions?  
Copilot is reading your code. Wait for some minutes, like this:  
![GIF](https://user-images.githubusercontent.com/70022443/151682285-56f82134-d12d-4d28-949b-c420aa4e80ef.gif)  
and the suggestions will appear.

**Reply by jekabso21** [reply]
I was waiting like a half an hour and nothing showed up

**Reply by katorly** [reply]
> I was waiting like a half an hour and nothing showed up

Are there same issues when typing other codes?

**Comment by davecheney** [comment]
Can you please past the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.
